react-hash-routing/client/components/header.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <header className="mb-5">
+      <nav className="navbar navbar-dark bg-dark shadow-sm">
+        <div className="container">
+          <div className="col px-0">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="" className="navbar-brand">
```

Comment:
Although this will work to navigate back home, it is better to use `href="#"`. By putting the hash there it shows it was intentional, versus leaving it empty looks like you forgot to put something there.